docs/section21/Rule21-11.md

```diff
@@ -0,0 +1,38 @@
+
+# Boiler - Rule 21-11  
+
+**Rule ID:** 21-11  
+**Rule Description:** When the system uses boilers the hot water system shall be modeled as primary only with continuous variable flow and a minimum turndown of 25%.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.5 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12
+2. B-RMR is not modeled with purchased heating.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12: `if PLACEHOLDER_AIRSIDE_SYSTEM_RULE-X-XX:`
+2. B-RMR is not modeled with purchased heating: `if NOT Rule-21-1:`
```

Comment:
each system type is likely to have its own rule. so this will bee a list of rules corresponding to each system type.

Reply:
Same comment as for other HW RDSs

Reply:
Updated.